WebContent/active/Unvalidated-Redirect/Redirect-FalsePositives-GET/inclusion-logic3.jsp

```diff
@@ -282,9 +282,19 @@
                     	}
                     					
                     	if (defaultInputType == DefaultInputType.RELATIVE_INPUT) {
+                            // This had previously been using request.getServerName().
+                            // However, that actually means that it 'trusts' the Host
+                            // header from the traffic, which turns this test which was
+                            // meant to be a FP case into a TP case for scenarios when
+                            // headers are being manipulated by a scanner.
+                            // There should only be one configured (default is localhost)
+                            MBeanServer mbs = ManagementFactory.getPlatformMBeanServer();
+                            Set<ObjectName> hosts = mbs.queryNames(new ObjectName("Catalina:type=Host,*"), null);
+                            String firstHostName = (String) mbs.getAttribute(hosts.get(0), "name");
+
                     		//relative to current dir path
                     		defaultBasePath = FileConstants.HTTP_PREFIX
-                           		+ request.getServerName() + ":" +
+                           		firstHostName + ":" +
```

Comment:
To maximize the logic/clarity of things we should probably change the AF plan to use localhost vs 127.0.0.1. (However, that's a separate repo/PR.)

Reply:
Or change this to use the address.

Reply:
We could leave the plans as 127.0.0.1, I don't think it breaks anything. It would just be more consistent to use localhost (IMHO).